src/main/java/world/bentobox/aoneblock/oneblocks/OneBlocksManager.java

```diff
@@ -174,6 +169,31 @@ void initBlock(String blockNumber, OneBlockPhase obPhase, ConfigurationSection p
             }
             addFirstBlock(obPhase, phase.getString(FIRST_BLOCK));
         }
+        // Icon block
+        if (phase.contains(ICON)) {
+            if (obPhase.getIconBlock() != null) {
+                throw new IOException("Block " + blockNumber + ": Icon block trying to be set to " + phase.getString(FIRST_BLOCK) + " but already set to " + obPhase.getFirstBlock() + " Duplicate phase file?");
+            }
+            ItemStack item;
+            try {
+                item = new ItemStack(Material.valueOf(phase.getString(ICON)), 1);
```

Comment:
Relying on an exception for code logic is a code smell and bad form. To find out if a Material exists or not, you can use this:

```
Material m = Enums.getIfPresent(Material.class, phase.getString(ICON).toUpperCase(Locale.ENGLISH)).orNull();
if (m == null) {
  item = tryPlayerHead();
} else {
  item = new ItemStack(m); // note that the default quantity is 1, so you don't need to say 1
}
```